src/AdoNet/Orleans.Statistics.AdoNet/MySQL-Statistics.sql

```diff
@@ -1,40 +0,0 @@
-CREATE TABLE OrleansStatisticsTable
```

Comment:
those are needed for other AdoNet tests :( I will add them back and rename them .

Reply:
sorry edit: those are needed for other AdoNet providers such as membership and persistence, Since liveness tests and persistence tests started failing and crying for this sql.

Reply:
or maybe just their tests,  I don't know. I will figure it out. no action item here. I'm just rambling on.

Reply:
These shouldn't be needed for anything... unless I removed something non-statistics related?

Reply:
sorry it was just one line you forget to remove in SqlServerStorageForTesting. and that caused tests failure because the test infrastructure is looking for it .

Reply:
Oh, thanks, Xiao

Reply:
```
protected override string[] SetupSqlScriptFileNames
        {
            get { return new[] {
                    "SQLServer-Main.sql",
                    "SQLServer-Clustering.sql",
                    "SQLServer-Persistence.sql",
                    "SQLServer-Reminders.sql",
                    "SQLServer-Statistics.sql"
                };
            }
        }
```
I removed it in my pr. sorry for the exaggeration. Since from first sight, it looks like the removal of those files caused test failure. While it really just caused the test infra to failed to start